src/mono/mono/mini/aot-runtime.c

```diff
@@ -940,6 +941,14 @@ typedef struct {
 	gboolean no_aot_trampoline;
 } MethodRef;
 
+static MonoMethod*
+inflate_unsafe_accessor_wrapper (MonoMethod *extern_decl, MonoGenericContext *ctx, MonoUnsafeAccessorKind accessor_kind, const char *member_name, MonoError *error)
```

Comment:
`inflate_unsafe_accessor_wrapper` is a sub-routine of `inflate_unsafe_accessor_like_decl`. `inflate_unsafe_accessor_like_decl` could be refactored with a call to `inflate_unsafe_accessor_wrapper`,